submit_cgap/tests/test_submission.py

```diff
@@ -298,7 +298,7 @@ def make_user_record(title='J Doe',
 def test_get_user_record():
 
     def make_mocked_get(auth_failure_code=400):
-        def mocked_get(url, *, auth):
+        def mocked_get(url, *, auth, **kwargs):
             ignored(url)
```

Comment:
You will almost surely need to mark these references to `kwargs` ignored to get past `make lint`.